scene/gui/rich_text_label.cpp

```diff
@@ -1881,7 +1910,10 @@ void RichTextLabel::_bind_methods() {
 	ADD_GROUP("BBCode", "bbcode_");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "bbcode_enabled"), "set_use_bbcode", "is_using_bbcode");
 	ADD_PROPERTY(PropertyInfo(Variant::STRING, "bbcode_text", PROPERTY_HINT_MULTILINE_TEXT), "set_bbcode", "get_bbcode");
+
+
```

Comment:
The double empty newline make the clang-format style checker complain, as our style guide prevents excessive whitespace: https://travis-ci.org/godotengine/godot/jobs/250104559

To avoid such issues, you should probably try to run clang-format on your changes locally before committing. You can do that with an IDE plugin, or directly with a pre-commit hook (see the hook we propose in `misc/hooks/`).